src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_delete_instances.py

```diff
@@ -0,0 +1,183 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss delete-instances",
+)
+class DeleteInstances(AAZCommand):
+    """Delete VMs within a VMSS.
+
+    :example: VirtualMachineScaleSet_DeleteInstances_MaximumSet_Gen
+        az Delete VMs within a VMSS. (autogenerated) az vmss delete-instances --instance-ids 0 --name MyScaleSet --resource-group MyResourceGroup
+    """
+
+    _aaz_info = {
+        "version": "2024-11-01",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.compute/virtualmachinescalesets/{}/delete", "2024-11-01"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, None)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`.",
+            required=True,
+        )
+        _args_schema.vm_scale_set_name = AAZStrArg(
+            options=["-n", "--name", "--vm-scale-set-name"],
```

Comment:
I think it’s better to stay consistent with other VMSS commands and the existing behavior of this command by using only the `-n` and `--name` aliases for this parameter.
```suggestion
            options=["-n", "--name"],
```